requirements/edx/base.txt

```diff
@@ -71,15 +71,15 @@ django-multi-email-field==0.5.1  # via edx-enterprise
 django-mysql==2.4.1
 git+https://github.com/edx/django-oauth-plus.git@b9b64a3ac24fd11f471763c88462bbf3c53e46e6#egg=django-oauth-plus==2.2.9.edx-4
 django-oauth-toolkit==1.1.3
-django-object-actions==1.1.2  # via edx-enterprise
+django-object-actions==2.0.0  # via edx-enterprise
```

Comment:
This release drops Python 2 support: https://github.com/crccheck/django-object-actions/blob/master/CHANGELOG.md .  Might be ok if we wait until we've rolled out Python 3 to merge this.